### PR TITLE
Updated docstrings in bci_data

### PR DESCRIPTION
**Pull request by anup2ladder**
Branched from `docstrings-classification`

Updated docstrings on `bci_data` module, which contains the `EEG_data` and `ERP_data` classes.

Filled in docstring information with what is there and put a template when information on information on descriptions, parameters or returns are unavailable. When information is missing, the template text is "Description of (param)" – thus, searching for the string "Description of" will work to locate where more information is needed.

### CODE REVIEW

File: bci_essentials/__init__.py [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+3. Provide options for visualizing BCI data
+
+Limitations
+-----------
+1. Currently will not handle ERP sessions longer that 10,000 markers in
+duration to avoid latency cause by dynamic sizing of numpy ndarrays.
+This number can be increased by changing the `max_windows variable` in
+the `ERP_data` class.
 
 """
```
Comment (danielcomaduran) [inline bci_essentials/__init__.py:25]: Minor thing, I would just code the `max_windows` not `max_windows` variable

File: bci_essentials/bci_data.py [hunk 2]
```diff
@@ -647,8 +787,44 @@ def mne_export_resting_state_as_raw(self):
     # SIGNAL PROCESSING
     # Preprocessing goes here (windows are nchannels by nsamples)
     def preprocessing(self, window, option=None, order=5, fc=60, fl=10, fh=50):
-        """
-        Howdy
+        """Signal preprocessing.
```
Comment (danielcomaduran) [inline bci_essentials/bci_data.py:790]: These values are missing the optional label to them

File: bci_essentials/bci_data.py [hunk 3]
```diff
@@ -667,8 +843,28 @@ def preprocessing(self, window, option=None, order=5, fc=60, fl=10, fh=50):
 
     # Artefact rejection goes here (windows are nchannels by nsamples)
     def artefact_rejection(self, window, option=None):
-        """
-        Howdy
+        """Artefact rejection.
```
Comment (danielcomaduran) [inline bci_essentials/bci_data.py:846]: Minor, optional is missing the **optional** label to it

Comment (anup2ladder) [reply bci_essentials/bci_data.py:846]: Fixed

File: bci_essentials/bci_data.py (hunk 2 above)
Comment (anup2ladder) [reply bci_essentials/bci_data.py:790]: Fixed

**Pull request merged**